index.js

```diff
@@ -239,98 +240,120 @@ module.exports = {
     return {
       post: {
         // Route to accept the submitted form.
-        submit: async function (req) {
-          const input = req.body;
-          const output = {};
-          const formErrors = [];
-          const formId = self.inferIdLocaleAndMode(req, req.body._id);
-
-          const form = await self.find(req, {
-            _id: self.apos.launder.id(formId)
-          }).toObject();
-
-          if (!form) {
-            throw self.apos.error('notfound');
-          }
+        submit: [
+          require('connect-multiparty')(),
```

Comment:
Anywhere we use this we need a try/finally around the whole route that deletes all of the temporary files for all properties of `req.files` at the end.